versions.gradle.kts

```diff
@@ -0,0 +1,28 @@
+buildscript {
+    // Library version
+    extra["publishVersion"] = "1.15.0"
```

Comment:
@lucasvalenteds sorry if this is a dumb question, but is this where you are setting all the values referenced in all the other build scripts?

Reply:
Yes, the `version.gradle.kts` file contains all variables that was defined in `build.gradle` before the rewrite.

The `extra` variable is equivalent to the `ext` property in Groovy DSL. It's a container to store data that can be shared across multiple files.

To share them, the function `apply(from = "version.gradle.kts")` file is called in the root build script (`build.gradle.kts`) and propagated to the subprojects using the `subprojects {}` block.